internal/lsp/hover.go

```diff
@@ -0,0 +1,174 @@
+package lsp
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/olekukonko/tablewriter"
+
```

Comment:
Is there a reason to break these groups up? If so, we might want to do the same in the server.go too for consistency.

Reply:
We do that in server.go too, don't we? :)

Imports are grouped according to this rule: https://github.com/StyraInc/regal/blob/main/docs/development.md#linting

Reply:
Ahh ok, so OPA is different from other external reps, gotcha.